packages/nuejs.org/docs/index.md

```diff
@@ -34,18 +34,18 @@ Nue is a great fit for the following people:
 
 
 ### Less but better code
-What used to take a UI designer, React engineer, and an absurd amount of JavaScript can now be done by a UX developer and just a handful of CSS:
+What used to take a UI designer, React engineer and an absurd amount of JavaScript code can now be done by a UX developer and just a handful of CSS:
 
 [image]
   small: /img/figma-to-css.png
   large: /img/figma-to-css-big.png
 
 
-With Nue, you can take advantage of a [global design system](global-design-system.html)  and [modern, minimalistic CSS](css-best-practices.html) to build websites with significantly less code than what you need with a JavaScript-based framework like **Next.js**. Your codebase can be so small, that it's in the ballpark of a [single Tailwind button](/blog/introducing-nue-css/). This changes the way you think about web development. Chances are that Nue turns you into a professional UX developer, who thinks more like a designer and less of an engineer.
+Nue allows you to take advantage of a [global design system](global-design-system.html)  and [modern, minimalistic CSS](css-best-practices.html) to build websites with significantly less code than with a JavaScript-based framework like **Next.js**. Your codebase can as small as a [single Tailwind button](/blog/introducing-nue-css/). This changes the way you think about web development. Chances are that Nue turns you into a professional UX developer, who thinks more like a designer and less than an engineer.
```

Comment:
```suggestion
Nue allows you to take advantage of a [global design system](global-design-system.html)  and [modern, minimalistic CSS](css-best-practices.html) to build websites with significantly less code than with a JavaScript-based framework like **Next.js**. Your codebase can as small as a [single Tailwind button](/blog/introducing-nue-css/). This changes the way you think about web development. Chances are that Nue turns you into a professional UX developer, who thinks more like a designer and less like an engineer.
```

Reply:
I've just noticed that we need "be" just before "as small as" in this sentence.
Your change from "than" to "like" is great.

Reply:
Good catch!